extruct/uniform.py

```diff
@@ -1,16 +1,27 @@
 from six.moves.urllib.parse import urlparse, urljoin
 
 
-def _uopengraph(extracted):
+def _uopengraph(extracted, with_og_array=False):
     out = []
     for obj in extracted:
         # In order of appearance in the page
         properties = list(reversed(obj['properties']))
-        # Ensuring that never empty value is returned if there is a duplicated
-        # property with non empty value
+        # Set of non empty properties
         non_empty_props = {k for k, v in properties if v and v.strip()}
```

Comment:
Super minor, but it seems that the code could be simplified if ``non_empty_props`` is removed and instead this condition is checked directly below, because we need one check per element in any case.

Reply:
Okay will handle this